src/components/TaskBody.vue

```diff
@@ -652,6 +653,27 @@ $breakpoint-mobile: 1024px;
 		opacity: .6;
 	}
 
+	&--non-started .title {
+		color: var(--color-placeholder-dark);
+		font-style: italic;
+	}
+
+	&--non-started .title:before {
+		content: '⏱️';
```

Comment:
Please use material design icon for this. Just put it next to the `Notes` and `Pin` icons here:
https://github.com/nextcloud/tasks/blob/00dc26d40aec882196a3cf56bd065d6000532e1f/src/components/TaskBody.vue#L77-L82

Reply:
The clock icon would be suitable, I think:
https://materialdesignicons.com/icon/alarm

Reply:
> Please use a material design icon for this. Just put it next to the `Notes` and `Pin` icons

Yes of course : 5243df0 ... Thanks for pointing me the right direction !

> The clock icon would be suitable, I think

If I may, I prefere the `calendar-clock`
![image](https://user-images.githubusercontent.com/93337483/168072366-90658827-8e10-4a7e-93ee-2754575d85fe.png)

I think it looks more consistent with the `calendar-start` used in the task settings :
![image](https://user-images.githubusercontent.com/93337483/168072457-26a75710-ae42-4e4e-8254-1956c51cfedd.png)

It looks like this (with the alt text) :
![2022-05-12_14-12-25](https://user-images.githubusercontent.com/93337483/168071994-9d735f38-e584-42f5-a922-8268fcb3f895.png)

Reply:
> > The clock icon would be suitable, I think
> 
> If I may, I prefere the `calendar-clock` ![image](https://user-images.githubusercontent.com/93337483/168072366-90658827-8e10-4a7e-93ee-2754575d85fe.png)

Fine with me.